src/query/expression/src/function.rs

```diff
@@ -591,7 +591,7 @@ impl<'a> EvalContext<'a> {
                     )
                 };
 
-                Err(ErrorCode::Internal(err_msg).set_span(span))
```

Comment:
Can we customize error code? default is BadArguments

Reply:
It requires some major refactor on `FunctionContext`. In what case custom error code is necessary?

Reply:
return `ErrorCode::BadArguments` is okay, as most errors in function are caused by bad arguments.